src/Documentation/tutorials_and_samples/tutorial_1_helloworld.md

```diff
@@ -0,0 +1,377 @@
+---
+layout: page
+title: Tutorial 1 Hello World
+---
+
+# Tutorial 1: Hello World
+
+This process recreates the same Hello World sample application available [here](https://github.com/dotnet/orleans/tree/master/Samples/2.0/HelloWorld).
+
+## Overview of the parts
+
+This application consists of a solution that contains four projects: SiloHost, OrleansClient, HelloWorld.Interfaces, and HelloWorld.Grains.
+We will create the SiloHost and OrleansClient projects as .NET Core App files and we will create the HelloWorld.Interfaces and HelloWorld.Grains projects as .NET Standard Libraries.
+Orleans comes into the picture when we add the NuGet packages to each of these four projects and replace the default code with the Orleans code provided in this tutorial.
+Finally, we will configure the dependency references among the projects before building and running the solution.
+
+## How the parts work together
+
+SiloHost is started first.
+Then, OrleansClient is started. OrleansClient tells the system to create a grain based on the grain interface provided.
```

Comment:
```suggestion
Then, OrleansClient is started. OrleansClient will creates a reference to a IHello grain,  and call its SayHello() method through its interface IHello. This programming model is built as part of our core concept of distributed Object Oriented Programming```

Reply:
I edited the file directly, rather than using the "Apply suggestion" button.

The final version of that section looks like this:

> SiloHost is started first.
> Then, OrleansClient is started.
> OrleansClient creates a reference to the IHello grain and calls its SayHello() method through its interface, IHello.
> This programming model is built as part of our core concept of distributed Object Oriented Programming.
> Next, the grain is activated in the silo.
> OrleansClient sends a greeting to the activated grain.
> The grain returns a response to OrleansClient, which OrleansClient displays on the console.

Reply:
that's ok. I was trying out the Suggestion feature, never seen it before :D